### ISSUE TITLE
After upgrade to 0.50.0 I get: " Extension of type 'VersionCatalogsExtension' does not exist "

### ISSUE THREAD

**Issue by jschneider** [jmfayard/refreshVersions#599]
- [ ] I have read the [guidelines for submitting issues](https://jmfayard.github.io/refreshVersions/contributing/submitting-issues/)
## 🐛 Describe the bug

I upgraded to 0.50.0 (from 0.40.2) by updating the version number in `settings.gradle.kts` and `buildSrc/settings.gradle.kts`

## ⚠️ Current behavior

`gradle refreshVersions` fails with this message:

```
> Task :refreshVersions FAILED

----------------------------------------------------------------------------------------------------------------------------------------------------
183 projects                                                                                 CONF        EXEC    TOT EXE FLD SKP UTD WRK CHD NSR ACT
----------------------------------------------------------------------------------------------------------------------------------------------------
: .............................................................................. FAILURE [  5.634 s] [  0.013 s]   1   1   1   0   0   1   0   0   1
----------------------------------------------------------------------------------------------------------------------------------------------------

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':refreshVersions'.
> Extension of type 'VersionCatalogsExtension' does not exist. Currently registered extension types: [ExtraPropertiesExtension, BasePluginExtension, DefaultArtifactPublicationSet, IdeaModel, NexusStagingExtension, KoverProjectConfig, KoverMergedConfig, ReportingExtension, LicensePluginExtension, NodeJsRootExtension, YarnRootExtension]
```

## 📱 Tech info

Outout: 
https://gist.github.com/jschneider/e7f2e61d72ddfa152e581f812d57ea52

Gradle: 7.4.2
Ubuntu 22.04
Java 17.0.4

 - [Build Scan URL](https://scans.gradle.com/s/zso35r7v54iri)

**Comment by LouisCAD** [comment]
Thanks for the report.

You can probably work it around by creating an empty `grade/libs.versions.toml` file for now.

I'll make a quick fix for that in 0.50.1

**Comment by jschneider** [comment]
I created the file:
`touch gradle/libs.versions.toml`

Seems work fine now! Thanks!

**Comment by LouisCAD** [comment]
Fixed in 0.50.1, thank you again for the prompt report!
